components/counter-bar/src/index.js

```diff
@@ -6,24 +6,23 @@ import ResultCountTitle from '@govuk-frederic/result-count-title';
 
 import { BLACK, GREY_1, GREY_3, LINK_COLOUR, RED, WHITE, YELLOW } from 'govuk-colours';
 
-const Container = styled('div')({
+const Wrapper = styled('div')({
   display: 'flex',
+  flexWrap: 'nowrap',
```

Comment:
is this really right? don't we want wrapping?

Reply:
I think so as this is what prevents the items container from wrapping under the total - it always remains on the right. However, the rule can be removed as it is the default!

Reply:
Feel free to run it, change that to `wrap` in the Inspector and see what you think if you like.

Reply:
ah - ok - seems fair enough - hadn't really groked that this works in conjunction with the other container